dev_guide/builds.adoc

```diff
@@ -710,10 +710,10 @@ https://developer.github.com/webhooks/[GitHub webhooks] or Generic webhooks.
 
 https://developer.github.com/webhooks/creating/[GitHub webhooks] handle the call
 made by GitHub when a repository is updated. When defining the trigger, you must
-specify a link:../dev_guide/secrets.html[`*secret*`] as part of the URL you supply
-to GitHub when configuring the webhook. The `*secret*` ensures that only you and
-your repository can trigger the build. The following example is a trigger
-definition JSON within the `*BuildConfig*`:
+specify a `*secret*`, which will be part of the URL you supply to GitHub when
+configuring the webhook. The `*secret*` ensures the uniqueness of aforementioned
+URL, iow. that only you and your repository can trigger the build. The following
```

Comment:
"The secret ensures the uniqueness of the URL, preventing others from triggering the build."